packages/@o3r/core/schematics/ng-add/schema.json

```diff
@@ -53,6 +53,12 @@
       "description": "Install otter configuration package and config extractor setup",
       "x-prompt": "Add Otter configuration setup ?"
     },
+    "enableRulesEngine": {
+      "type": "boolean",
+      "default": true,
```

Comment:
hmmm I would set `false` per default, I think the biggest part of the apps does not need a rules-engine from the beginning

Reply:
ok, I see